cypress/integration/registrationFlows/emailRegistration.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Sign up with email', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.visit('/enter?state=new-user');
+  });
+
+  // Regression test for #11099
+  it('should preserve the form data on unsuccesful submissions', () => {
+    cy.get('a').contains('Sign up with Email').click();
+
+    cy.get('input[type="file"]').attachFile('images/admin-image.png');
```

Comment:
Can we target this one by the label text too? It would make the test more specific, and helps us check the form is accessible too 
```suggestion
    cy.findByLabelText('whatever the input's label or aria-label is').attachFile('images/admin-image.png');
```